src/dg/weak_dg.cpp

```diff
@@ -578,6 +578,8 @@ void DGWeak<dim,nstate,real,MeshType>::assemble_volume_term_explicit(
     std::vector< ADArrayTensor1 > conv_phys_flux_at_q(n_quad_pts);
     std::vector< ADArrayTensor1 > diss_phys_flux_at_q(n_quad_pts);
     std::vector< doubleArray > source_at_q(n_quad_pts);
+    //adding physical source
+    std::vector< doubleArray > physical_source_at_q(n_quad_pts);
```

Comment:
General comment, this should also be implemented in `strong_dg.cpp`

Reply:
fixed